src/services/dashboards/widgets/_components/WidgetDataTable.vue

```diff
@@ -159,21 +178,18 @@ export default defineComponent<Props>({
             type: Number,
             default: 1,
         },
-        pageSize: {
-            type: Number,
-            default: 5,
+        showLegend: {
+            type: Boolean,
+            default: false,
+        },
+        showLegendIndex: {
+            type: Boolean,
+            default: false,
         },
```

Comment:
Is there index number in design?
If not, I think it can be deleted for now. There's too many props 🥲

Reply:
It exists in the design specification, so I think we need to add it.🥲